src/formatters/default_formatter.erl

```diff
@@ -1342,6 +1352,45 @@ lay_application(Name, Arguments, SpacesWithinParentheses, Ctxt) ->
             maybe_parentheses(D, Prec, Ctxt)
     end.
 
+%% @doc Might produce a new AST on which the functions in the export list
+%%      are sorted depending on what 'sort_exported_funcs' was set to:
+%%          - alphabetically, if set to 'alphabetically'
+%%          - left as it is, if set to 'false'
+sort_exported_funcs([{tree, list, Attrs, {list, Funcs0, none}}] = Arguments, SortFun) ->
```

Comment:
Can we use `erl_syntax` functions instead of pattern-matching on the tuple representation here and everywhere else? Those are supposed to be records, after all.